CHANGELOG.md

```diff
@@ -4,6 +4,72 @@
 
 # JupyterLab Changelog
 
+## v4.4
+
+JupyterLab 4.4 includes a number of new features (described below), bug fixes, and enhancements.
+This release is compatible with extensions supporting JupyterLab 4.0.
+Extension authors are recommended to consult the [Extension Migration Guide](https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#jupyterlab-4-3-to-4-4) which lists deprecations and changes to the public API.
+
+### Code console improvements
+
+TODO
+
+### Settings import and export
+
+Settings can now be exported to `overrides.json` from the Settings Editor,
+which can be used to pre-configure defaults on deployments or for restoring
+the settings.
+
+### Workspace indicator
+
+An opt-in workspace indicator displayed on the top bar (next to the menu bar) can be enabled
+in the Settings → View → Appearance → Show Workspace Indicator.
+
+### Support for collaboration without `RTC:` drive
+
+Real time collaboration in previous versions of JupyterLab
+(which can be enabled by installing jupyter-collaboration package)
+was implemented by swapping the default JupyterLab file system
+drive to a collaborative drive with the `RTC:` prefix.
+This was not compatible with multiple extensions.
+JupyterLab 4.4 introduces Content Providers API which will
+enable the next version of `jupyter-collaboration` to work
+without changing the drive prefix, and thus improves
+compatibility with extensions which do not support arbitrary drives.
+
+### Kernel subshells support
+
+Kernel subshells, introduced in [JEP 91](https://jupyter.org/enhancement-proposals/91-kernel-subshells/kernel-subshells.html),
+enable concurrent code execution, in kernels which support them.
+When performing a long-running computation (such as training of a model), subshells enable:
+- using `ipywidgets` with updates displayed immediately,
+- monitoring the kernel resource usage (e.g. memory usage),
+- invoking arbitrary commands, e.g. to continue improving code to be used in a subsequent
+  cell, while having live access to the previously defined variables.
+
+Subshells are supported by `ipykernel` 7 (currently in alpha).
+
+To create a subshell, open context menu (right-click)
+over a notebook using a kernel supporting subshells
+and select "New Subshell Console for Notebook".
+A new console panel will open allowing access to variables from the main session.
+
+To enable the use of subshell by widgets (such as `ipywidgets`)
+change the Kernel Comms over subshells setting.
```

Comment:
TBC, pending https://github.com/jupyterlab/jupyterlab/pull/17363

Reply:
Should we keep it here, or wait until #17363 is merged to add it back later in a separate PR?

Reply:
Let's drop it and add back in #17363 itself.

```suggestion
```